api/oauth2.js

```diff
@@ -54,6 +54,7 @@ module.exports.load = async function(app, db) {
 
       if (scopes.replace(/identify/g, "") == scopes) missingscopes.push("identify");
       if (scopes.replace(/email/g, "") == scopes) missingscopes.push("email");
+      if (newseetings.api.client.j4r.enabled) if (scopes.replace(/guilds/g, "") == scopes) missingscopes.push("email");
```

Comment:
`missingscopes.push("email");` would be incorrect. Make it into `missingscopes.push("guilds");`.

Reply:
is this a question or for me to change it because i changed it

Reply:
1. "newseetings" should be "newsettings".
2. `missingscopes.push("email");` should change into `missingscopes.push("guilds");`
3. Did you test your changes?